archive/app/controllers/ArchiveController.scala

```diff
@@ -64,7 +64,7 @@ class ArchiveController(dynamoDB: DynamoDB) extends Controller with Logging with
         Some(path)
       case _ => None
     }
-    s"http://${normalised.getOrElse(path)}"
+    s"https://${normalised.getOrElse(path)}"
```

Comment:
Does this mean we input and store urls with the scheme but actually ignores when returning the actual redirect? If yes this is quite confusing